stock_no_negative/models/stock_quant.py

```diff
@@ -12,6 +12,10 @@ class StockQuant(models.Model):
 
     @api.constrains("product_id", "quantity")
     def check_negative_qty(self):
+        # If there is stock.move for subcontracting receipts, the context will be passed
+        # from mrp_subcontracting_skip_no_negative and pass the check for finished products.
```

Comment:
Can we adjust the comment to avoid limiting the expected usage to the subcontracting receipt scenario? I'm not sure myself there is any other case where people might want to use this context, but let's keep the possibility open.

```suggestion
        # To provide an option to skip the check when necessary. e.g. mrp_subcontracting_skip_no_negative
        # passes the context for subcontracting receipts.
```